packages/vite-plugin-cloudflare/package.json

```diff
@@ -40,9 +40,10 @@
 		"test:ci": "vitest run"
 	},
 	"dependencies": {
+		"@cloudflare/unenv-preset": "~1.1.1",
```

Comment:
- Not sure if this should be "1.1.1" (no `~`)? We can relax the range once 2.0 is released
- Is it expected that the plugin has no (peer) dep on `workerd`?

Reply:
wrangler is in both devDep and peerDep

Reply:
Wrangler is there twice because the `devDep` is only for testing (and we use the workspace version); while the peerDep is a range ("wrangler": "^3.101.0") which constrains the minimum version.

Reply:
the cf unenv preset has its own peer dependency on workerd.

Reply:
> Happy to pin this dep to 1.1.1 if that helps

It's more that we do not expect any 1.1.2...

> the cf unenv preset has its own peer dependency on workerd.

it's a peer dep so it should be installed somewhere...

Reply:
It'll be installed as part of Wrangler